make_script.R

```diff
@@ -0,0 +1,19 @@
+library(dplyr)
+library(readr)
+library(stringr)
+library(sbtools)
+library(whisker)
+
+source("./1_fetch/src/fetch_data.R")
+source("./2_process/src/process_data.R")
+source("./3_visualize/src/plot_rmse.R")
+
+# Get the data from ScienceBase
+fetch_data(save_path <- "./1_fetch/out",output_file_name <- "model_RMSEs.csv")
```

Comment:
I'm curious: does this work with `<-` for argument assignment? I thought functions needed to use `=` for arguments